### PR TITLE
[Bug 1145671] Use ES 1.2.4 in Travis (as well as 0.90.10)

### PR DESCRIPTION
**Pull request by mythmon**
Lets see if it is really just easy.

### CODE REVIEW

File: scripts/travis/setup.sh [hunk 1]
```diff
@@ -48,9 +48,15 @@ echo "Updating product details"
 python manage.py update_product_details
 
 echo "Starting ElasticSearch"
-pushd elasticsearch-0.90.10
-  # This will daemonize
-  ./bin/elasticsearch
+pushd elasticsearch-${ELASTICSEARCH_VERSION}
+  # New version of ES are foreground by default, old ones are backgrounded by default
```
Comment (willkg) [inline scripts/travis/setup.sh]: I <3 this fun change in ES.

**Comment by willkg** [comment]
Travis passed this. Looks good to me. r+

File: scripts/travis/setup.sh (hunk 1 above)
Comment (mythmon) [inline scripts/travis/setup.sh]: I think it is a good change to make, it is kind of surprising when you run something big like ES and it just does... nothing (apparently). Plus, running in foreground by default helps with things like init systems and blah blah blah.

I just hate straddling the change like this.

**Comment by mythmon** [comment]
Thanks!

**Pull request merged**